test/remote/fedex_test.rb

```diff
@@ -391,7 +391,7 @@ def test_obtain_shipping_label_with_label_format_option
 
     assert response.success?
     refute_empty response.labels
-    data = response.labels.first.img_data 
+    data = response.labels.first.img_data
     refute_empty data
     assert data[0...4] == '%PDF'
```

Comment:
Feel free to keep a note for later, but given that `nil` doesn't have `empty?`, I think we could simplify all of this to `assert_equal '%PDF', response.labels.first.img_data[0...4]`